Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'turbolinks'
 gem 'jbuilder', '~> 2.0'
 
 gem "cssbundling-rails", "~> 1.1"
-gem "primer_view_components", "~> 0.1.0"
+gem "primer_view_components", "~> 0.14.0"
```

Comment:
We have no visual regression testing for this app -- did you get a chance to boot it up locally and check the admin panel for anything obviously broken?

Reply:
Yup! Didn't set the PR to "ready" until we gave it a tire-kick in stage... splits are loading fine on the summary and detail view and I was able to add a user to a split manually w/o error in the controller action or background job. 😄